pkg/note/note.go

```diff
@@ -115,29 +133,29 @@ func rsaKeyHash(name string, key *rsa.PublicKey) (uint32, error) {
 // NewNoteSigner converts a sigstore/sigstore/pkg/signature.Signer into a note.Signer.
 func NewNoteSigner(ctx context.Context, origin string, signer signature.Signer) (note.Signer, error) {
 	if !isValidName(origin) {
-		return &noteSigner{}, fmt.Errorf("invalid name %s", origin)
+		return nil, fmt.Errorf("invalid name %s", origin)
 	}
 
 	pubKey, err := signer.PublicKey()
 	if err != nil {
-		return &noteSigner{}, fmt.Errorf("getting public key: %w", err)
+		return nil, fmt.Errorf("getting public key: %w", err)
 	}
 	var keyID uint32
 	switch pk := pubKey.(type) {
 	case *ecdsa.PublicKey:
 		keyID, err = ecdsaKeyHash(pk)
```

Comment:
unlike the other keys, cdsaKeyHash() does not take `origin` into account. I don't have the context so don't know if this is a mistake or not -- what is origin?